deployable/module-template/variables.tf

```
@@ -0,0 +1,14 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
```

```suggestion
  description = "The IBM Cloud API key."
```

Can we add what it's used for? Or that it has to match the key for other executable modules, or ...?